### PR TITLE
fix the link to hook

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: zhouhao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

**Comment by q384566678** [comment]
updated.

File: runtime.md [hunk 1]
```diff
@@ -137,4 +137,4 @@ Once a container is deleted its ID MAY be used by a subsequent container.
 
 ## <a name="runtimeHooks" />Hooks
 Many of the operations specified in this specification have "hooks" that allow for additional actions to be taken before or after each operation.
-See [runtime configuration for hooks](./config.md#hooks) for more information.
+See [runtime configuration for hooks](./config.md#posix-platform-hooks) for more information.
```
Comment (wking) [inline runtime.md]: While we're touching this, I'd drop the useless `./` prefix.  But I'm fine with this PR landing without that change.

**Comment by hqhq** [comment]
LGTM

**Comment by q384566678** [comment]
ping @crosbymichael

**Comment by crosbymichael** [comment]
LGTM

Thanks @q384566678

**Pull request merged**